### ISSUE TITLE
Build failure on Fedora 30

### ISSUE THREAD

**Issue by djberg96** [rwmjones/libnbd#1]
Just for kicks, tried to build libnbd on my local Fedora 30 VM, and hit this:

```
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -Dsysconfdir=\"/usr/local/etc\" -pthread -I/usr/include/libxml2 -g -O2 -MT libnbd_la-crypto.lo -MD -MP -MF .deps/libnbd_la-crypto.Tpo -c crypto.c  -fPIC -DPIC -o .libs/libnbd_la-crypto.o
crypto.c:672:1: error: conflicting types for ‘nbd_internal_crypto_create_session’
  672 | nbd_internal_crypto_create_session (struct nbd_handle *h)
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from crypto.c:33:
internal.h:237:23: note: previous declaration of ‘nbd_internal_crypto_create_session’ was here
  237 | extern struct socket *nbd_internal_crypto_create_session (struct nbd_handle *, struct socket *oldsock);
      |                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
make[3]: *** [Makefile:650: libnbd_la-crypto.lo] Error 1
make[3]: Leaving directory '/home/dberger/Dev/libnbd/lib'
make[2]: *** [Makefile:504: all] Error 2
make[2]: Leaving directory '/home/dberger/Dev/libnbd/lib'
make[1]: *** [Makefile:501: all-recursive] Error 1
make[1]: Leaving directory '/home/dberger/Dev/libnbd'
make: *** [Makefile:428: all] Error 2
```

**Comment by ebblake** [comment]
fixed in dad3422c

**Comment by djberg96** [comment]
@ebblake Did you push that up? I'm not seeing it.

**Comment by ebblake** [comment]
https://github.com/libguestfs/libnbd/commit/dad3422c08c08ec68e960d6d7fd2d67bf8293389
Not yet released; will be in 0.1.3. It may be that you are looking at Rich's repo rather than the canonical upstream

**Comment by djberg96** [comment]
Oops, sorry, closing.
